client/client.go

```diff
@@ -198,6 +202,15 @@ func NewClientWithOpts(ops ...Opt) (*Client, error) {
 		}
 	}
 
+	// TODO: Maybe filter the _ping endpoint from otel tracing?
+	client.Transport = otelhttp.NewTransport(
```

Comment:
Will the extra wrapping affect things like `client.Close()` https://github.com/moby/moby/blob/96b473a0bdc3f4c1cd0324f3ff044b04ab349df8/client/client.go#L218-L221


And `client.Dialer()` ? https://github.com/moby/moby/blob/96b473a0bdc3f4c1cd0324f3ff044b04ab349df8/client/client.go#L352

Reply:
Good catch... we need to fix those.

Reply:
Fixed.